lib/src/common.d.ts

```diff
@@ -1,6 +1,7 @@
 export declare let oDataQueryNames: string[];
 export declare const DEFAULT_VERSION = "v1.0";
 export declare const GRAPH_BASE_URL = "https://graph.microsoft.com/";
+export declare const PACKAGE_VERSION = "1.0.0";
```

Comment:
Added the comment:
// NOTE: The PACKAGE_VERSION version here should match exactly the package version in package.json.